beginner_source/text_sentiment_ngrams_tutorial.py

```diff
@@ -156,188 +186,211 @@ def generate_batch(batch):
 #
 
 
-######################################################################
-# `torch.utils.data.DataLoader <https://pytorch.org/docs/stable/data.html?highlight=dataloader#torch.utils.data.DataLoader>`__
-# is recommended for PyTorch users, and it makes data loading in parallel
-# easily (a tutorial is
-# `here <https://pytorch.org/tutorials/beginner/data_loading_tutorial.html>`__).
-# We use ``DataLoader`` here to load AG_NEWS datasets and send it to the
-# model for training/validation.
-#
-
-from torch.utils.data import DataLoader
+import time
```

Comment:
perhaps can be imported in next code snippet as it might not be used here?

Reply:
It's used in L195.